.github/workflows/ci.yml

```diff
@@ -76,6 +83,16 @@ jobs:
         ARROW_PRE_0_15_IPC_FORMAT: 1
       run: |
         coverage run --source=coffea/ setup.py pytest
+    - name: Test WQ
+    if: matrix.os != 'windows-latest' && matrix.python-version == 3.6
```

Comment:
@zsurma this if statement needs two more spaces. Yaml is whitespace delimited so this is breaking the script which is why CI isn't running!

Reply:
@lgray thanks for finding that, my editor sometimes auto-formats in a weird way and I didn't catch it that time!

Reply:
and can you change `matrix.python-version == 3.6` to `matrix.python-version != 3.8`, just so we test 3.7 and only exclude 3.8. Or is there a problem with 3.7 as well?

Reply:
@lgray I'll change that over now - we're noticing some issues with the way the environment is being created in the other versions too so trying to fix that as well